### PR TITLE
fix (refs T30974): edit variables naming for box-shadow.

### PR DESCRIPTION
**Pull request by ahmad-demos**
https://yaits.demos-deutschland.de/T30974

- Edit variables naming for box-shadow.
- Edit variables naming for rounded.
- Edit variables naming for font-size.

### CODE REVIEW

File: demosplan/DemosPlanCoreBundle/Resources/client/scss/token/index.scss [hunk 1]
```diff
@@ -8,6 +8,6 @@
 //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 // Each file defines token variables.
-@import 'box-shadow';
+@import '@demos-europe/demosplan-ui/tokens/scss/_boxShadow';
```
Comment (spiess-demos) [inline demosplan/DemosPlanCoreBundle/Resources/client/scss/token/index.scss:11]: I'd suggest to import those tokens directly in `DemosPlanCoreBundle/Resources/client/scss/core_style.scss:44`, so we can drop the `/scss/token/` folder entirely in the end.

Another point: when importing the scss partial, you don't need the underscore:

```scss
@import '@demos-europe/demosplan-ui/tokens/scss/boxShadow';
```

should work as well.

**Review by spiess-demos — CHANGES_REQUESTED** [review]

File: demosplan/DemosPlanCoreBundle/Resources/client/scss/token/index.scss (hunk 1 above)
Comment (ahmad-demos) [reply demosplan/DemosPlanCoreBundle/Resources/client/scss/token/index.scss]: Done in https://github.com/demos-europe/demosplan-core/pull/1307/commits/a8203013c01b2cd5cd370001f81d7830b1187fdb, thanx

**Review by spiess-demos — APPROVED** [review]
lgtm!

File: demosplan/DemosPlanCoreBundle/Resources/client/scss/utility/_font-size.scss [hunk 2]
```diff
@@ -11,62 +11,62 @@
 
 // 50px
 .font-size-brand-hero {
```
Comment (spiess-demos) [inline demosplan/DemosPlanCoreBundle/Resources/client/scss/utility/_font-size.scss:13]: To be in Line with the other utility classes (dropping the "domain" part of their names), i'd suggest to change it like:

```suggestion
.font-size-hero {
```

**Review by spiess-demos — CHANGES_REQUESTED** [review]
one concern only

File: demosplan/DemosPlanCoreBundle/Resources/client/scss/token/index.scss [hunk 3]
```diff
@@ -8,6 +8,4 @@
 //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 // Each file defines token variables.
-@import 'box-shadow';
 @import 'font-size';
```
[the remark this answers is not in the document]
Comment (ahmad-demos) [reply demosplan/DemosPlanCoreBundle/Resources/client/scss/token/index.scss]: Done in https://github.com/demos-europe/demosplan-core/pull/1307/commits/b72844aed17cc85d71836c2778735849f42769ba, thanx!

**Review by spiess-demos — APPROVED** [review]
lgtm (code review). 
however, after merging this into the integration branch, i gonna have a final look at the comment on top of `core_style.scss`.

**Pull request merged**